.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "mavenlint-react",
   "env": {
+    "browser": true,
```

Comment:
I think we don't want to do this. Almost left a note about this, but we need our components to run on the server or the client, and `window` stuff in our components is a big no-no for that.

What we can do is have the `styleguide` directory have it's own eslint config that allows browser stuff